client/question/question.js

```diff
@@ -93,6 +93,38 @@ tie.constant('CLASS_NAME_AUXILIARY_CODE', 'AuxiliaryCode');
  */
 tie.constant('CLASS_NAME_SYSTEM_CODE', 'System');
 
+/**
+ * Name of button that will direct users to Python language primer.
+ *
+ * @type {string}
+ * @constant
+ */
+tie.constant('PYTHON_PRIMER_BUTTON_NAME', 'New to Python?');
+
+/**
+ * Constant to represent the consecutiveLanguageUnfamiliarityCounter option
+ * for the
+ *
+ * resetCounters function in FeedbackGeneratorService. Is to be passed into the
+ * resetCounters function as a parameter when we want to reset all counters
+ * except the consecutiveLanguageUnfamiliarityCounter.
+ *
+ * @type {string}
+ * @constant
+ */
+tie.constant('EXCLUDE_LANGUAGE_UNFAMILARITY_ERROR',
```

Comment:
ERROR_COUNTER_LANGUAGE_UNFAMILIARITY
ERROR_COUNTER_SAME_RUNTIME

("error counter" is what it is, and the suffix is what type of error counter it is. Starting with 'exclude' makes it sound like a verb)